packages/cli-dashboard/src/components/siteReport/tabs/index.ts

```diff
@@ -37,7 +37,7 @@ const Tabs: SidebarItems = {
     dropdownOpen: true,
   },
   [SIDEBAR_ITEMS_KEYS.COOKIES_WITH_ISSUES]: {
-    title: 'Cookie Issues',
+    title: 'cookieIssues',
```

Comment:
For later refactoring, since this is the title key instead of the actual title string now, the key name should reflect that.